app/components/primer/alpha/action_list/item.rb

```diff
@@ -235,8 +238,7 @@ def initialize(
         def before_render
           @system_arguments[:classes] = class_names(
             @system_arguments[:classes],
-            "ActionListItem--withActions" => trailing_action.present?,
-            "ActionListItem--navActive" => active?
+            "ActionListItem--withActions" => trailing_action.present?
```

Comment:
How does this change impact NavList?

Reply:
`NavList` adds this class so it's not necessary for `ActionList` to do it.